casanovo/denovo/dataloaders.py

```diff
@@ -122,6 +126,17 @@ def setup(self, stage: str = None, annotated: bool = True) -> None:
             )
             if self.test_index is not None:
                 self.test_dataset = make_dataset(self.test_index)
+        if stage in (None, "db"):
```

Comment:
I don't think you should have 'None' here. My understanding is that if no stage is specified in the config, the model will prepare train, val, and test datasets. I think it should only prepare a db search dataset if specifically requested to do so, since it is not part of the standard casanovo workflow.